docs/en/core-services/Caching.md

```diff
@@ -49,8 +49,8 @@ The number of data replicas within Alluxio storage is determined dynamically by
 Due to the fact that Alluxio relies on the under file storage for a majority of data storage,
 Alluxio does not need to keep copies of data that are not being used.
 
-Alluxio also supports tiered storage, which makes the system storage media aware, enabling data
-storage optimizations similar to L1/L2 CPU caches.
+Alluxio also supports tiered storage configurations making the storage system media aware.
+This enables decreased fetching latencies similar to how L1/L2 CPU caches operate.
```

Comment:
I feel this can be further elaborated to make it easier to fully understand. For example: `This enables decreased fetching latencies similar to how L1/L2 CPU caches operate. For example if you have 3 tiers MEM, SSD and HDD, reading cach in the MEM tier is faster than SSD and HDD.` What do you think?

Reply:
updated